packages/dev/buildTools/src/copyAssets.ts

```diff
@@ -49,14 +51,38 @@ export const processAssets = (options: { extensions: string[] } = { extensions:
                 ignoreInitial: false,
                 awaitWriteFinish: {
                     stabilityThreshold: 1000,
-                    pollInterval: 200,
+                    pollInterval: 300,
                 },
                 alwaysStat: true,
                 interval: 300,
                 binaryInterval: 600,
             })
             .on("all", (_event, file) => {
+                // don't track directory changes
+                if (_event === "addDir" || _event === "unlinkDir") {
+                    return;
+                }
+                // be sure that no file is processed twice at the same time
+                if (currentlyProcessing.includes(file)) {
+                    verbose && console.log(`Already processing asset: ${file}, event: ${_event}`);
+                    return;
+                }
+                currentlyProcessing.push(file);
+                let verb: string = "";

```

Comment:
If the intent here is to handle all `_event` types other than `"addDir"` and `"unlinkDir"`, then it would be more type safe to not initialize `verb`. That way, if you missed one of the event types in the switch block or if a new event type got added in a future version of the library, line 85 would be a compile error (`Variable 'file' is used before being assigned.(2454)`) and you'd know about the problem right away. Otherwise it would be harder to diagnose.